extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -10126,6 +10143,31 @@
 	'this_page' => 'esta página',
 	'top_five' => 'top cinco',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "top cinco" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)